app/services/staff_nomis_checker.rb

```diff
@@ -56,7 +56,7 @@ def offender
   end
 
   def prisoner_restrictions
-    if Nomis::Feature.offender_restrictions_info_enabled?(@visit.prison_name) &&
+    if Nomis::Feature.restrictions_info_enabled?(@visit.prison_name) &&
         offender.valid?
```

Comment:
Perhaps outside the scope of this PR but wonder if we need consistency around terms 'prisoner' vs 'offender' - it feels like we are randomly using one or the other

Reply:
Will add a ticket